### DISCUSSION TITLE
Configurable Redirect URI for logout endpoint

### DISCUSSION THREAD

**Discussion by kakatam** [keycloak/keycloak#8709] [Ideas]
Currently, if a user logs out from Keycloak Account Management page, then the redirect_uri of the logout url is always /auth/realms/<realm-id>/account. I can imagine that this is not always desirable as you might want to redirect your users back to your application. So, I would like to propose to extend the client settings for which you can configure a redirect URI for the logout endpoint.

**Comment by stianst** [comment]
By "back to your application" are you talking about when an application has linked to the account console, and there's a link to go back to the application?
